ospfd/ospf_interface.c

```diff
@@ -1454,13 +1454,14 @@ void ospf_reset_hello_timer(struct interface *ifp, struct in_addr addr,
 	struct route_node *rn;
 
 	if (is_addr) {
-		struct prefix p;
```

Comment:
just use struct prefix?  Why the change?  Especially since ospf_if_table_lookup is using struct prefix

Reply:
Thanks!
I have correct it. Seems i had done redundant things.